### PR TITLE
Fixes within instructional documents

### PR DESCRIPTION
**Pull request by pixelstuermer**
I recently wanted to contribute to some translations/typos/grammar mistakes but I first encountered problems/questions with the instruction documents.

Therefore I first updated the `README`, `CONTRIBUTING` and `TRANSLATION_NOTES` documents. I also changed one format from markdown to asciidoc to maintain the continuity of the files and their syntax (especially because asciidoc is explicitly recommended).

- I fixed some typos
- I corrected and added some (external) links
- I fixed some asciidoc formatting/syntax
- I removed some duplicate text, especially within the readme file where some passages were duplicates
- I removed some diff information, for example "<<<<<<< HEAD"

### CODE REVIEW

File: CONTRIBUTING.asc [hunk 1]
```diff
@@ -0,0 +1,15 @@
+= Zur Übersetzung der zweiten Auflage von Git Pro beitragen
+
+== Ein Problem melden
+
+Bevor Sie ein Problem melden, bitten wir Sie zu überprüfen, ob sich nicht ein ähnliches oder gar dasselbe Problem bereits im Bugtracking-System befindet.
+
+Wenn das Problem auch auf der link:https://git-scm.com[Git-Seite] zu finden ist, überprüfen Sie bitte auch, ob das Problem dort noch aktuell ist. Es könnte schon erledigt, aber noch nicht veröffentlicht sein.
```
Comment (mynetx) [inline CONTRIBUTING.asc]:
```suggestion
Wenn das Problem auch auf der link:https://git-scm.com[Git-Website] zu finden ist, überprüfen Sie bitte auch, ob das Problem dort noch aktuell ist. Es könnte schon erledigt, aber noch nicht veröffentlicht sein.
```

File: README.asc [hunk 2]
```diff
@@ -1,39 +1,22 @@
...
+= Pro Git: Zweite Auflage
 
-Herzlich Willkommen bei der zweiten Ausgabe des Buches Pro Git.
+Herzlich willkommen bei der zweiten Auflage des Buchs Pro Git.
 
-Sie finden dieses Buch in Zukunft online unter: http://git-scm.com/book/de/v2
+Sie finden dieses Buch in Zukunft online unter: https://git-scm.com/book/de/v2
 
-Wie die Erste, ist auch die zweite Auflage von Pro Git Open Source und steht unter der Creative Commons Lizenz.
+Wie die erste, ist auch die zweite Auflage von Pro Git Open Source und steht unter der Creative Commons Lizenz.
```
Comment (mynetx) [inline README.asc]:
```suggestion
Wie die erste, ist auch die zweite Auflage von Pro Git Open Source und steht unter der Creative Commons-Lizenz.
```

Let’s put a hyphen here.

File: README.asc [hunk 3]
```diff
@@ -1,39 +1,22 @@
...
-
-Statt einem großen Repository für alle Sprachen, wird jede Sprache jetzt in einem einzelnen Repository verwaltet. Im Dokument link:CONTRIBUTING.md[Hilfe bei der deutschen Übersetzung] findet man weitere Informationen.
-
-Einige Dinge haben sich seit der Öffnung der ersteN Edition geändert.
-Zum einen haben wir für das zweite Buch von Markdown zum großartigen ASCII-Format gewechselt. 
-
-Ausserdem halten wir die Übersetzungen des Buches jetzt in eigenen Repositories, anstatt in Unterordnern des Haupt-Repositorys.
-Mehr Informationen finden Sie unter link:CONTRIBUTING.md[the Contributing document].
+Im Vergleich zur ersten Auflage haben sich in der zweiten Auflage allerdings ein paar Dinge geändert: Zum einen wird jetzt statt Markdown das AsciiDoc-Format verwendet und zum anderen wird O'Reilly's Buildserver unter der link:https://atlas.oreilly.com[Atlas Platform] genutzt. Mit Hilfe von Continuous Integration wird das Buch in allen Sprachen erstellt und zur Verfügung gestellt.
```
Comment (mynetx) [inline README.asc]:
```suggestion
Im Vergleich zur ersten Auflage haben sich in der zweiten Auflage allerdings ein paar Dinge geändert: Zum einen wird jetzt statt Markdown das AsciiDoc-Format verwendet und zum anderen wird O’Reilly-Buildserver unter der link:https://atlas.oreilly.com[Atlas Platform] genutzt. Mit Hilfe von Continuous Integration wird das Buch in allen Sprachen erstellt und zur Verfügung gestellt.
```

In German, we don’t use the genitive apostrophe.

File: README.asc [hunk 4]
```diff
@@ -1,39 +1,22 @@
...
 
+Statt einem großen Repository für alle Sprachen, wird jede Sprache mittlerweile in einem eigenen Repository verwaltet. Im Dokument link:TRANSLATION_NOTES.asc[Hilfe bei der deutschen Übersetzung] findet man weitere Informationen.
```
Comment (mynetx) [inline README.asc]:
```suggestion
Statt eines großen Repositorys für alle Sprachen wird jede Sprache mittlerweile in einem eigenen Repository verwaltet. Im Dokument link:TRANSLATION_NOTES.asc[Hilfe bei der deutschen Übersetzung] finden Sie weitere Informationen.
```

`Statt` is followed by genitive; let’s address the reader directly.

File: README.asc [hunk 5]
```diff
@@ -50,15 +33,12 @@ Converting to PDF...
...
-=======
-== Ein Problem berichten
+== Ein Problem melden
 
-Bevor Sie ein Problem melden, bitten wir Sie zu überprüfen, ob sich nicht ein ähnliches oder gar das selbe bereits im Bugtracking-System befindet.
+Bevor Sie ein Problem melden, bitten wir Sie zu überprüfen, ob sich nicht ein ähnliches oder gar dasselbe Problem bereits im Bugtracking-System befindet.
 
-Wenn das Problem auch auf der git-scm.com Seite zu finden ist, überprüfen Sie bitte auch, ob das Problem hier noch aktuell ist.
-Es könnte schon erledigt, aber noch nicht veröffentlicht sein.
+Wenn das Problem auch auf der link:https://git-scm.com[Git-Seite] zu finden ist, überprüfen Sie bitte auch, ob das Problem dort noch aktuell ist. Es könnte schon erledigt, aber noch nicht veröffentlicht sein.
```
Comment (mynetx) [inline README.asc]:
```suggestion
Wenn das Problem auch auf der link:https://git-scm.com[Git-Website] zu finden ist, überprüfen Sie bitte auch, ob das Problem dort noch aktuell ist. Es könnte schon erledigt, aber noch nicht veröffentlicht sein.
```

see above

File: TRANSLATION_NOTES.asc [hunk 6]
```diff
@@ -1,61 +1,54 @@
-== Hinweise zur deutschen Übersetzung
+= Hinweise zur deutschen Übersetzung
 
-<<<<<<< HEAD
-In diesem Dokument werden alle Punkte rund um die deutsche Übersetzung gesammelt. In diesem Dokument möchten wir festlegen, ob der Leser mit Du oder Sie angesprochen wird, ob der Erzähler die 1. Person Singular oder die 1. Person Plural verwendet, oder wie technische Fachbegriffe übersetzt werden sollen.
-=======
-After forking this repository to translate the work, this file is where the notes for coordinating the translation work would go.
-Things like standardizing on words and expressions so that the work is consistent or notes on how the contributing process is to be handled.
->>>>>>> 811dffebd4ff1348819f8acc70f954bd4ad65057
+In diesem Dokument werden alle Informationen rund um die deutsche Übersetzung gesammelt. Hiermit möchten wir beispielsweise festlegen, ob der Leser mit "Du" oder "Sie" angesprochen wird, ob der Erzähler die 1. Person Singular oder die 1. Person Plural verwendet oder wie technische Fachbegriffe übersetzt werden sollen.
```
Comment (mynetx) [inline TRANSLATION_NOTES.asc:3]:
```suggestion
In diesem Dokument werden alle Informationen rund um die deutsche Übersetzung gesammelt. Hiermit möchten wir beispielsweise festlegen, ob der Leser mit „Du“ oder „Sie“ angesprochen wird, ob der Erzähler die 1. Person Singular oder die 1. Person Plural verwendet oder wie technische Fachbegriffe übersetzt werden sollen.
```

I opt for typographical quotes – would like to get feedback on this. It’s also worth adding a note about our decision to the relevant section on the book’s style guide for German.

File: TRANSLATION_NOTES.asc [hunk 7]
```diff
@@ -1,61 +1,54 @@
...
 
-<<<<<<< HEAD
-=== Workflow Git
+== Workflow Git
 
-Wenn Sie an der deutschen Übersetzung mitarbeiten wollen, können Sie dazu ein Fork erstellen und in diesem weiterarbeiten.
+Wenn Sie an der deutschen Übersetzung mitarbeiten wollen, können Sie dazu ein link:https://git-scm.com/book/de/v1/Distribuierte-Arbeit-mit-Git-xxx-An-einem-Projekt-mitarbeiten#Kleine,-öffentliche-Projekte[Fork] erstellen und in diesem weiterarbeiten.
 
-* Bitte erstellen Sie einen Pull-Request, wenn Sie ein Arbeitspaket abgeschlossen haben. Bitte beschreiben Sie im Pull-Request was Ihr zu mergender Branch enthält (Neue Übersetzung, Korrektur, ...).
+* Bitte erstellen Sie erst einen Pull-Request, wenn Sie ein Arbeitspaket abgeschlossen haben. Bitte beschreiben Sie im Pull-Request, was Ihr zu mergender Branch enthält (neue Übersetzungen, Korrekturen, etc.).
```
Comment (mynetx) [inline TRANSLATION_NOTES.asc]:
```suggestion
* Bitte erstellen Sie erst einen Pull-Request, wenn Sie ein Arbeitspaket abgeschlossen haben. Bitte beschreiben Sie im Pull-Request, was Ihr zu mergender Branch enthält (neue Übersetzungen, Korrekturen usw.).
```

`etc` or `usw`? Also, I don’t think we use a comma before this ellipsis abbreviation.

File: TRANSLATION_NOTES.asc [hunk 8]
```diff
@@ -1,61 +1,54 @@
...
 
-* Wir werden Ihr Ergebnis prüfen bzw. ein weiterer Helfer wird Ihr Ergebnis Korrektur lesen (Review). Das kann dazu führen, dass Sie Ihre Arbeit noch einmal überarbeiten müssen. Bitte sehen Sie das Review als positive Hilfestellung damit das Ergebnis insgesamt besser wird und nehmen Sie die Kritik nicht negativ auf. Wir wollen damit sicherstellen, dass die deutsche Übersetzung einheitlicher wird und in einer guten Qualität zur Verfügung steht. Wenn alles passt, nehmen wir das Ergebnis in den Hauptzweig auf und veröffentlichen es für die bekannten Seiten.
+* Wir werden Ihren Beitrag prüfen und ein weiterer Helfer wird Ihr Ergebnis Korrektur lesen (Review). Das kann dazu führen, dass Sie Ihre Arbeit noch einmal überarbeiten müssen. Bitte sehen Sie das Review als positive Hilfestellung, damit das Ergebnis insgesamt besser wird und nehmen Sie die Kritik nicht negativ auf. Wir wollen damit sicherstellen, dass die deutsche Übersetzung einheitlicher wird und in einer guten Qualität zur Verfügung steht. Wenn alles passt, nehmen wir das Ergebnis in den Hauptzweig auf und veröffentlichen es für die bekannten Seiten.
```
Comment (mynetx) [inline TRANSLATION_NOTES.asc]:
```suggestion
* Wir werden Ihren Beitrag prüfen und ein weiterer Helfer wird Ihr Ergebnis Korrektur lesen (Review). Das kann dazu führen, dass Sie Ihre Arbeit noch einmal überarbeiten müssen. Bitte sehen Sie das Review als positive Hilfestellung, damit das Ergebnis insgesamt besser wird, und nehmen Sie die Kritik nicht negativ auf. Wir wollen damit sicherstellen, dass die deutsche Übersetzung einheitlicher wird und in einer guten Qualität zur Verfügung steht. Wenn alles passt, nehmen wir das Ergebnis in den Hauptzweig auf und veröffentlichen es für die bekannten Seiten.
```

Comma required here.
Also, the last bit sounds a bit unclear – what are `known sides/pages`?

File: TRANSLATION_NOTES.asc [hunk 9]
```diff
@@ -1,61 +1,54 @@
...
+* Kommandozeilenausgaben sollten so übersetzt werden, dass sie mit der deutschen Version von Git übereinstimmen. Im Zweifel belassen Sie bitte die Kommandozeilenausgabe in Englisch.
 
-=== Allgemeine Regeln
+== Allgemeine Regeln
 
-* Der Leser wird formal mit "Sie" angesprochen und das "Sie" wird auch groß geschrieben. Bitte beachten Sie dies auch bei Possessivpronomen, wie z.B. Ihre oder Ihre. Siehe hierzu auch http://www.duden.de/sprachwissen/sprachratgeber/gross-oder-kleinschreibung-von--em-du-du--em--und--em-ihr-ihr--em--1. Andere Sprachen verwenden auch die formelle Form, siehe hierzu https://github.com/progit/progit2/issues/151.
+* Der Leser wird formal mit "Sie" angesprochen, wobei das "Sie" auch großgeschrieben wird. Bitte beachten Sie dies auch bei Possessivpronomen, wie beispielsweise "Ihr", "Ihre", etc. Siehe hierzu auch link:http://www.duden.de/sprachwissen/sprachratgeber/gross-oder-kleinschreibung-von--em-du-du--em--und--em-ihr-ihr--em--1[folgender Link]. Andere Sprachen verwenden ebenfalls die formelle Form, wie link:https://github.com/progit/progit2/issues/151[hier] beschrieben.
```
Comment (mynetx) [inline TRANSLATION_NOTES.asc]:
```suggestion
* Der Leser wird formal mit „Sie“ angesprochen, wobei das „Sie“ auch großgeschrieben wird. Bitte beachten Sie dies auch bei Possessivpronomen, wie beispielsweise „Ihr“, „Ihre“ usw. Siehe hierzu auch link:http://www.duden.de/sprachwissen/sprachratgeber/gross-oder-kleinschreibung-von--em-du-du--em--und--em-ihr-ihr--em--1[folgender Link]. Andere Sprachen verwenden ebenfalls die formelle Form, wie link:https://github.com/progit/progit2/issues/151[hier] beschrieben.
```

see above – requesting feedback on this.

File: TRANSLATION_NOTES.asc [hunk 10]
```diff
@@ -1,61 +1,54 @@
...
 
-=== Schreibweise und Übersetzung von Fachbegriffen
+== Schreibweise und Übersetzung von Fachbegriffen
 
-Die Übersetzungen orientieren sich an der deutschen Übersetzung der Programmdatei von git (/usr/bin/git bzw. git.exe).
+Die Übersetzungen orientieren sich an der deutschen Übersetzung der Programmdatei von git (`/usr/bin/git` beziehungsweise `git.exe`).
 
-Wenn ein Fachbegriff in der Liste fehlt, bitte prüfen, ob dieser in der Git-Programmdatei verwendet wird (siehe hierzu https://github.com/git/git/blob/master/po/de.po). 
+Wenn ein Fachbegriff in dieser Liste fehlt, prüfen Sie bitte, ob dieser in der Git-Programmdatei verwendet wird (siehe hierzu link:https://github.com/git/git/blob/master/po/de.po[folgender Link]).
```
Comment (mynetx) [inline TRANSLATION_NOTES.asc]:
```suggestion
Wenn ein Fachbegriff in dieser Liste fehlt, überprüfen Sie bitte, ob dieser in der Git-Programmdatei verwendet wird (siehe hierzu link:https://github.com/git/git/blob/master/po/de.po[folgender Link]).
```

File: TRANSLATION_NOTES.asc [hunk 11]
```diff
@@ -1,61 +1,54 @@
...
-==== A - D
+=== A – D
 
 [width="100%", frame="topbot", options="header,footer"]
 |==============================================================================
 |Englisch|Deutsch
 |Branch|
-Branch; Singular: der Branch; Plural: die Branches; Alternativ kann auch die
-deutsche Übersetzung "Zweig" verwendet werden
+Branch; Singular: der Branch; Plural: die Branches; Alternativ kann auch die deutsche Übersetzung "Zweig" verwendet werden
```
Comment (mynetx) [inline TRANSLATION_NOTES.asc]:
```suggestion
Branch; Singular: der Branch; Plural: die Branches; Alternativ kann auch die deutsche Übersetzung „Zweig“ verwendet werden
```

see above, feedback required before deciding on quote style

File: TRANSLATION_NOTES.asc [hunk 12]
```diff
@@ -113,23 +105,17 @@ Zum Stash hinzufügen
...
 
-=== Als Maintainer helfen
+== Als Maintainer helfen
 
-Wenn Sie nicht nur übersetzen möchten, sondern uns auch bei der Koordination unterstützen wollen, dann melden Sie sich bitte bei einem Maintainer.
-=======
-As the work is translated, please update the `status.json` file to indicate the rough percentage complete each file is.
-This will be shown on various pages to let people know how much work is left to be done.
->>>>>>> 811dffebd4ff1348819f8acc70f954bd4ad65057
+Wenn Sie nicht nur zur Übersetzug beitragen möchten, sondern uns auch bei der Koordination unterstützen wollen, dann melden Sie sich bitte bei einem Maintainer.
```
Comment (mynetx) [inline TRANSLATION_NOTES.asc]:
```suggestion
Wenn Sie nicht nur zur Übersetzung beitragen möchten, sondern uns auch bei der Koordination unterstützen wollen, dann melden Sie sich bitte bei einem Maintainer.
```

**Review by mynetx — COMMENTED** [review]
Corrected typos, some suggestions that we might want to discuss

**Comment by mynetx** [comment]
@pixelstuermer Thanks Philipp for your thorough look through these important documents. Let’s optimise them a bit more and let me know what you think about the quote style.

@fernsehkind Your feedback is appreciated.

**Comment by pixelstuermer** [comment]
Thanks @mynetx for your support. I guess the quoting is fine since it‘s German ;)

Thanks a lot again.

**Comment by mynetx** [comment]
@pixelstuermer Hmmm, maybe you are able to [squash your commits](https://blog.github.com/2016-04-01-squash-your-commits/) so the history is a bit cleaner? But we can also do this upon merging the PR.

**Comment by pixelstuermer** [comment]
Yep, I just did @mynetx 👍

**Comment by pixelstuermer** [comment]
@mynetx do you have write access?

**Comment by mynetx** [comment]
@pixelstuermer I will take a look shortly, and yes :)

**Comment by pixelstuermer** [comment]
Great, thanks 👍

**Pull request merged**